src/mslice/plotting/plot_window/slice_plot.py

```diff
@@ -29,6 +29,8 @@
 
 from typing import Callable
 
+from plotting.plot_window.cachable_input_dialog import QCacheableInputDialog
```

Comment:
```suggestion
from mslice.plotting.plot_window.cachable_input_dialog import QCacheableInputDialog
```
On Windows, MSlice could not find QCacheableInputDialog when using `plotting.plot_window.cachable_input_dialog import QCacheableInputDialog`